### PR TITLE
docs(react-drawer): fix drawer resizing storybook example

### PR DESCRIPTION
**Pull request by cperezruiz-microsoft**
## Previous Behavior

The storybook example of a drawer component that can be resized (linked here: https://react.fluentui.dev/iframe.html?viewMode=docs&id=components-drawer--default#resizable) does not actually resize. Most of the styling and functionality are present, but the html setup does not create a region that can be dragged, meaning the resizing logic is never performed.

Animation2

## New Behavior

I added a div container and a few extra CSS properties to allow the drawer's border to be draggable, and therefore resizable.

See working example here: https://stackblitz.com/edit/mw9wzn?file=src%2Fexample.tsx

Animation

### CODE REVIEW

**Comment by cperezruiz-microsoft** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/Avatar-mount_base.html">694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/Avatar-mount.html">646</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/Button-mount_base.html">320</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/Button-mount.html">314</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/Field-mount_base.html">1102</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/Field-mount.html">1152</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/FluentProvider-mount_base.html">699</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/FluentProvider-mount.html">732</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/FluentProviderWithTheme-mount_base.html">84</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/FluentProviderWithTheme-mount.html">96</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">33</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">36</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">86</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">83</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/MakeStyles-mount_base.html">850</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/MakeStyles-mount.html">855</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/Persona-mount_base.html">1818</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/Persona-mount.html">1731</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/SpinButton-mount_base.html">1412</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/SpinButton-mount.html">1390</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/SwatchPicker-mount_base.html">1557</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31372/perf-test-react-components/SwatchPicker-mount.html">1592</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-drawer/stories/Drawer/DrawerResizable.stories.tsx [hunk 1]
```diff
@@ -18,6 +18,15 @@ const useStyles = makeStyles({
     display: 'flex',
```
Comment (fabricteam) [inline packages/react-components/react-drawer/stories/Drawer/DrawerResizable.stories.tsx:18]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31372&commitId=0611a6860371eeb3572e4bdc70e1c0b584d26e5a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by marcosmoura — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Pull request merged**